source/jormungandr/jormungandr/parking_space_availability/bss/bss_provider_manager.py

```diff
@@ -49,6 +49,14 @@ def handle_places(self, places):
                 place['poi'] = self.handle_poi(place['poi'])
         return places
 
+    def handle_journeys(self, journeys):
+        # iterate through each 'from' and 'to' place for each section
+        for place in (s[from_to] for j in journeys for s in j['sections'] for from_to in ('from', 'to') if from_to in s):
```

Comment:
`for place in (s.get(from_to, {}) for j in journeys for s in j.get('sections', []) for from_to in ('from', 'to'))`